bindings/go/constructor/construct.go

```diff
@@ -56,10 +56,20 @@ func (c *DefaultConstructor) Construct(ctx context.Context, constructor *constru
 
 	for i, component := range constructor.Components {
 		componentLogger := logger.With("component", component.Name, "version", component.Version)
-		componentLogger.Info("constructing component")
+		componentLogger.Debug("constructing component")
 
 		eg.Go(func() error {
+			if c.opts.OnStartComponentConstruct != nil {
```

Comment:
Do you think we could have multiple of these ala chained observers?

Reply:
then you could have one OnStartComponentConstruct that delegates to multiple others right?